website/content/api-docs/system/inspect/router.mdx

```diff
@@ -0,0 +1,97 @@
+---
+layout: api
+page_title: /sys/internal/inspect/router - HTTP API
+description: >-
+  The '/sys/internal/inspect/router' endpoint focuses on viewing the contents of specific structures in the internal router subsystem.
+---
+
+# `/sys/raw`
+The `/sys/internal/inspect/router` is intended to inspect a specific internal subsytem for debugging purposes.
+It can be accessed with a root token or sudo priveleges.
```

Comment:
```suggestion
The `/sys/internal/inspect/router` is intended to inspect a specific internal subsystem for debugging purposes.
It can be accessed with a root token or sudo privileges.
```